packages/fxa-settings/src/components/Settings/PageRecoveryKeyCreate/index.test.tsx

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render, fireEvent, screen, waitFor } from '@testing-library/react';
+import * as Metrics from '../../../lib/metrics';
+
+import PageRecoveryKeyCreate from '.';
+
+describe('PageRecoveryKeyCreate', () => {
+  let logPageViewEventSpy: jest.SpyInstance;
```

Comment:
Just an observation - I hadn't noticed that the Settings pages use a different pattern  (spy) to test metrics than the new pages we're porting over from content-server.

Reply:
Yeah it threw me a little bit -- I was trying to recreate the existing tests for the page, but maybe it would be better to further the pattern we've set for the new react pages...

Reply:
Updated!